wkcuber/cubing.py

```diff
@@ -210,13 +210,19 @@ def cubing_job(args):
 
 
 def cubing(source_path, target_path, layer_name, dtype, batch_size, args) -> dict:
-
     source_files = find_source_filenames(source_path)
 
     # All images are assumed to have equal dimensions
     num_x, num_y = image_reader.read_dimensions(source_files[0])
     num_channels = image_reader.read_channel_count(source_files[0])
-    num_z = len(source_files)
+    num_z_slices_per_file = image_reader.read_z_slices_per_file(source_files[0])
```

Comment:
Does each file have the same number of z_slices or is there only a single file with many z_slices?

Reply:
As mentioned in the comment above, it is assumed that all images have equal dimensions, so I assume that for the number of z slices as well.